app/controllers/application_controller.rb

```diff
@@ -1,9 +1,27 @@
+
+
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def  aaa
-      1 + 1
+  def  aaa (a,b,c, d , e ,      f , g , h)     
+          1 + 1
+    e = 1 * 2 ** 3
+    puts [ 1, 2, 3 ].size
```

Comment:
Do not compute the size of statically sized objects.<br>Inconsistent indentation detected.<br>Space inside square brackets detected.